pkg/yurthub/kubernetes/serializer/serializer.go

```diff
@@ -39,27 +43,211 @@ import (
 	"k8s.io/klog"
 )
 
+var (
+	cacheCRDRESTMapperKey = "_internal/cache-manager/cache-crd-restmapper.conf"
+	sepForGVR             = "/"
+)
+
 // YurtHubSerializer is a global serializer manager for yurthub
 var YurtHubSerializer = NewSerializerManager()
 
-// UnsafeDefaultRESTMapper is only used to check whether the GVK is in the scheme according to the GVR information
-var UnsafeDefaultRESTMapper = NewDefaultRESTMapperFromScheme()
+// YurtHubRESTMapperManager is a global RESTMapper for yurthub
+var YurtHubRESTMapperManager = NewRESTMapperManager()
+
+// DynamicRESTMapper is a mapper that can dynamically update the mapping relationship between GVK and GVR
+type DynamicRESTMapper struct {
+	sync.Mutex
+	// storageEnabled is used to determine whether to enable hard cache
+	storageEnabled bool
+	// Used to hard cache the resourceToKind
+	storage        storage.Store
+	resourceToKind map[schema.GroupVersionResource]schema.GroupVersionKind
+}
+
+func NewDynamicRESTMapper() *DynamicRESTMapper {
+	m := make(map[schema.GroupVersionResource]schema.GroupVersionKind)
+	s, err := factory.CreateStorage()
```

Comment:
do not create Storage here, how about use the `StorageWrapper` from  `YurtHubConfiguration`?

Reply:
I combined RESTMapperManager with CacheManager, and the StorageWrapper of CacheManager will be used